docs/snippets/all/views/map.py

```diff
@@ -0,0 +1,20 @@
+"""Use a blueprint to show a map."""
+
+import rerun as rr
+import rerun.blueprint as rrb
+
+rr.init("rerun_example_gps_coordinates", spawn=True)
+
+rr.log("points", rr.GeoPoints([[47.6344, 19.1397], [47.6334, 19.1399]]))
+
+# Create a map view to display the chart.
+blueprint = rrb.Blueprint(
+    rrb.MapView(
+        origin="points",
+        name="MapView",
+        options=rrb.archetypes.MapOptions(provider=rrb.components.MapProvider.MapboxStreets),
```

Comment:
mapbox needs `RERUN_MAPBOX_ACCESS_TOKEN` env var, right? So this should probably either not use that or at least document this fact.
Also, we should ideally use all types here, so this should also show how to set the zoom level

Reply:
I'd like to keep that for follow-up PRs. I have these issues that I consider part of the 1st release MVP:
- https://github.com/rerun-io/rerun/issues/7903
- https://github.com/rerun-io/rerun/issues/7849

Reply:
👍